uncontrast_studies/processors/experiments_comparison.py

```diff
@@ -244,12 +276,10 @@ def aggregate_query_by_breakdown(self):
         experiment_ids = []
         for sig_option in significance_options:
             experiments_for_option = (
-                self.experiments.filter(significance=sig_option).distinct().values_list("id", flat=True)
+                self.experiments.filter(significance=sig_option).distinct()  # .values_list("id", flat=True)
```

Comment:
why?

Reply:
because it was redundant, happened also later